js/app.js

```diff
@@ -81,10 +81,30 @@ TextApp.prototype.setTheme = function() {
   this.editor_.setTheme(theme);
 };
 
+/**
+ * remove the editor so it can be reinitalised
+ */
+TextApp.prototype.removeEditor = function() {
+  const editor = $('#editor')[0];
```

Comment:
nb i was trying to move away from jquery use in the hopes i would get a chance to delete it one day. fine if you want to use it though

Reply:
i just copied and pasted this from another part of the codebase, i am personally against jquery too so happy to change this